process/smartContract/process.go

```diff
@@ -903,10 +917,7 @@ func (sc *scProcessor) isSCExecutionAfterBuiltInFunc(
 	}
 
 	scExecuteOutTransfer := outAcc.OutputTransfers[0]
-	callType := determineCallType(tx)
-	txData := prependCallbackToTxDataIfAsyncCallBack(scExecuteOutTransfer.Data, callType)
-
-	function, arguments, err := sc.argsParser.ParseCallData(string(txData))
+	function, arguments, err := sc.argsParser.ParseCallData(string(scExecuteOutTransfer.Data))
```

Comment:
Is this a case of backwards compatibility?

Reply:
yes it is. this code was not enabled yet.